linux_os/guide/services/ntp/chronyd_run_as_chrony_user/bash/shared.sh

```diff
@@ -0,0 +1,9 @@
+# platform = Red Hat Enterprise Linux 7,multi_platform_fedora,Red Hat Enterprise Linux 8
+
+if grep -q 'OPTIONS=.*' /etc/sysconfig/chronyd; then
+	# trying to solve cases where the parameter after OPTIONS
+	#may or may not be enclosed in quotes
+	sed -i -E 's/^([\s]*OPTIONS=["]?[^"]*)("?)/\1 -u chrony\2/' /etc/sysconfig/chronyd
```

Comment:
What happens if there is already a `-u` option? Will it be duplicated?

Reply:
Yes it will and it does not matter, as the last option is honored by chrony.

Reply:
The remediation is not idempotent, if it is run multiple times, it can endup with multiple `-u` options.

Would it work if you captured any `-u \S+rege` and didn't put it back?